nservicebus/upgrades/6to7/index.md

```diff
@@ -166,6 +166,13 @@ The [Message Property Encryption feature](/nservicebus/security/property-encrypt
 See the [NServiceBus.Encryption.MessageProperty upgrade guide](/nservicebus/upgrades/externalize-encryption.md) for more details.
 
 
+## Json serialization
+
+The [Json serializer](/nservicebus/serialization/json/index.md) has been moved from the NServiceBus package. Use the external Json serializer available as a separate NuGet package, [NServiceBus.Newtonsoft.Json ](https://www.nuget.org/packages/NServiceBus.Newtonsoft.Json /).
```

Comment:
> The [Json serializer](/nservicebus/serialization/json/index.md) has been moved from the NServiceBus package

technically this did not happen in v7. in v7 the change is "we removed the merged json serializer" but the alternative externalone has existed for years